.gitignore

```diff
@@ -0,0 +1,16 @@
+.idea/
+*.iml
+.DS_Store
+
+.installed.cfg
+bin
+develop-eggs
+dist
+downloads
+eggs
+parts
+*.egg-info
+lib
+lib64
+*.pyo
+*.pyc
```

Comment:
Ok so there's a lot going on that I don't understand about this. I see that it seems to be folders? A few of these look like we're creating extensions? I'm not too sure. So my best guess is that we're describing what the .gitignore extension does? What does the " * " mean on line 15 and 16?